File: HackerRank-10-Days-Statistics/Day 5/Day_5_Poisson_Distribution_II.cpp
#include <cmath>
#include <cstdio>
#include <vector>
#include <iostream>
#include <algorithm>
#include<iomanip>
using namespace std;


int main() {
    /* Enter your code here. Read input from STDIN. Print output to STDOUT */ 
    float a,b;
    cin>>a>>b;
    
    double C_A= 160 + 40*(a+ a*a);
    double C_B= 128 + 40*(b+ b*b);
    cout<<fixed<<setprecision(3)<<C_A<<endl;
    cout<<fixed<<setprecision(3)<<C_B<<endl;
    return 0;
}

File: HackerRank-10-Days-Statistics/Day 5/Day_5_Poisson_Distribution_II.py
# Enter your code here. Read input from STDIN. Print output to STDOUT
lamda1,lamda2=list(map(float,input().strip().split()))

print(round(160+40*(lamda1 + lamda1**2),3))
print(round(128+40*(lamda2 + lamda2**2),3))